tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Build a Local Solochain
+description: Learn to compile and launch a local blockchain node using Polkadot-SDK. Build, run, and interact with a pre-configured node template.
+---
+
+# Build a Local Solochain
+
+## Introduction
+
+[Polkadot-SDK](https://github.com/paritytech/polkadot-sdk){target=\_blank} offers a versatile and extensible blockchain development framework, enabling you to create custom blockchains tailored to your specific application or business requirements. This tutorial guides you through compiling and launching a single local blockchain node using the [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank}.
```

Comment:
I'd like to see a bit of an explanation of solochains and why someone would want to build a solochain. I know solochain is pretty self-explanatory, but this is on the Polkadot docs where the main premise of the chains in Polkadot is that they're interconnected, so I would like to see it explicitly mentioned that this chain is not a parachain and it is not connected to a relay chain or other parachains

Reply:
This is a good idea  - but maybe does not belong here, but rather in some introductory / index page

Reply:
I don't know how much content we would have to describe what a solo chain is. I wasn't thinking it would be anything long. I'd really just like to see one sentence here or maybe alter the last sentence to be something like

"In this tutorial, you’ll compile and launch a single local blockchain node using the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=_blank}, giving you a fully functional chain that operates independently, without being connected to a relay chain or parachain."